### ISSUE TITLE
On 2.0 when printer restart it load 1.0 settings and reset EEPROM

### ISSUE THREAD

**Issue by luc-github** [luc-github/Repetier-Firmware-0.92#100]
reproduced with fw compiled on Windows 10 + arduino 1.6.5 and Windows 7 + 1.5.8

**Comment by luc-github** [comment]
If shutdown printer and wait few sec then restart and load fail safe - EEPROM is back to normal 
new restart do not reset to 1.0

**Comment by luc-github** [comment]
now it is back to normal - need to investigate

**Comment by luc-github** [comment]
Early today did not got this issue - then I flashed FW as 1.0 and as AiO for testing and load fale safe  to be in situation.
My guess is after that SDCard locked EEPROM.bin file  on SDCard in some way as there is no way 2.0 has 1.0 settings in FW, and because it use virtual EEPROM in memory then in SD Card the changes done in memory were never flushed to SD Card.
I did several restart using emergency stop but this did not unlocked the file as printer was still powered so each time EEPROM loaded was 1.0 as 2.0 was not properly set.

So power off unlocked the file and allowed to reset it properly, I remember I saw some patch submission lately  on arduino github - I will check - looks very similar to a previous issue https://github.com/luc-github/Repetier-Firmware-0.92/issues/59
